feat: added CHANGELOG.md file (#51)
italia/publiccode.yml: CHANGELOG.md

# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.2] - 2019-03-11
### Added

> - Country Extension

Reviewer: ```suggestion
- A new `countryExtensionVersion` key was added under each country-specific extension, in order to separate their versioning from the core.
```

> - 5 more entries in the `standalone` key

Reviewer: ```suggestion
- The `standalone` value for the `softwareType` key was deprecated in favor of more specific values: `standalone/desktop`, `standalone/web`, `standalone/backend`, `standalone/other`
```

> - `categories-list` file with new categories 

Reviewer: I would only list changes to the standard, rather than the structure of the documentation:

```suggestion
```

> - `scope-list` file with new categories 

Reviewer: ```suggestion
```

> - `intendedAudience/scope` key

Reviewer: ```suggestion
- `intendedAudience/onlyFor` was renamed to `intendedAudience/scope (with a different dictionary of values).
```

> - country.rst dedicated

Reviewer: I wouldn't log changes to the structure of the documentation but only to the spec:

```suggestion
```

### Changed

> - `tags` key -> `categories`

Reviewer: ```suggestion
- `tags` was replaced by`categories` (with a different dictionary of values).
```

> - ISO 639-2 standard for languages -> BCP 47

Reviewer: ```suggestion
- BCP47 is now used for languages instead of ISO 639-2, thus keys under `description` will now look like `en` instead of `eng`
```

> - `publiccode-yaml-version` -> `publiccodeYmlVersion`

Reviewer: Wasn't this changed in 0.1?

Author: No, 0.1 still has the old structure. This is the 0.1 tagged commit: https://github.com/italia/publiccode.yml/commit/09dcde97dcee24f6c2d7d9393e62d4fece23914f

- All files ported to RST from previous MD

> - Updated README file

Reviewer: ```suggestion
```

> - Updated rules on `contact`

Reviewer: ```suggestion
- `maintainance/contacts` is now mandatory only if `maintainance/type` is `internal` or `community`.
```

### Removed

> - `pa-types` file 

Reviewer: ```suggestion
```

> - `intendedAudience/onlyFor` key

Reviewer: ```suggestion
- The `freeTags` key was removed.
```
